### PR TITLE
Feature/add data page #349

### PR DESCRIPTION
**Pull request by izuku-franck1555**

### CODE REVIEW

**Comment by andykitch** [comment]
Hey Franck, looks like this pull request has my user role request work in it too... Once my pull request is merged in I'll have a proper look at this.

File: src/UI/components/datahub/dataHubPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import React from 'react';
+import { Grid } from '@mui/material';
+import Image from 'next/image';
+import Link from 'next/link';
+
+function DataHubPanel(props: any) {
+  return (
+    <div>
+      <main>
+        <Container>
```
Comment (andykitch) [inline src/UI/components/datahub/dataHubPanel.tsx:11]: You shouldn't need `div, main, Container` here - they're all on the dataHub page

File: src/UI/pages/dataHub.tsx [hunk 2]
```diff
@@ -16,7 +16,13 @@ function DataHub() {
         >
           <SectionPanel title="Data Hub Page">
             <AuthWrapper role="">
-              <DataHubPanel />
+              <div>
+                <main>
+                  <Container>
```
Comment (andykitch) [inline src/UI/pages/dataHub.tsx:21]: We don't need these here either - they're in lines 9-11 already

Comment (izuku-franck1555) [reply src/UI/pages/dataHub.tsx]: Sorry Andrew, it's ok now. Please, can you review ?

**Review by andykitch — APPROVED** [review]

**Pull request merged**